new-design-workflow.dot

```diff
@@ -0,0 +1,94 @@
+digraph {
```

Comment:
Bit off topic, but how did you create this?

Reply:
[Graphviz](https://graphviz.org/)'s `dot` CLI tool. It's awesome. I'll add a comment that describes how to re-build the image from source.